lib/fastly.js

```diff
@@ -67,37 +57,30 @@ program
   .alias(`pk`)
   .arguments(`<key>`)
   .action((key) => {
-    co(function *() {
-      if (!program.apikey) {
-        program.apikey = yield coPrompt(`${fastlyApiKeyDescription}: `);
-      }
-      if (!program.serviceid) {
-        program.serviceid = yield coPrompt(`${fastlyServiceIdDescription}: `);
-      }
-      const fastly = require(`fastly`)(program.apikey);
-
-      if (program.hardpurge) {
-        fastly.purgeKey(program.serviceid, key, handleFastlyResponse(`Purged key: ${key}`));
-      }
-      else {
-        fastly.softPurgeKey(program.serviceid, key, handleFastlyResponse(`Purged key: ${key}`));
-      }
-    });
+    if ((!fastlyApiKeyPresent(program)) || (!fastlyServiceIdPresent(program))) {
```

Comment:
This condition should work correctly without having each statement in its own parentheses. The way it is now is a little hard to read, in my opinion. You could safely replace this line with:
```
if (!fastlyApiKeyPresent(program) || !fastlyServiceIdPresent(program)) {
```